src/server.js

```diff
@@ -29,13 +28,11 @@ function sniffMethod(label, fn) {
             });
 
             if (callback) {
-                return callback.apply(self, arguments);
+                return callback.apply(this, arguments);
```

Comment:
ESLint się nie czepia? :)

Reply:
czepia się tak samo, jak czepiał się w oryginale na użycie `this`, ale eliminujemy brzydkie `that/self=this` :)

Reply:
Zastosowałem ten zabieg, aby zredukować liczbę błędów ESLinta

Reply:
w takim razie proponuję `/* eslint-disable no-invalid-this */` - zmiana ilości wystąpień w tym przypadku niewiele zmienia.

Reply:
Dobry pomysł. Dodasz?

Reply:
added https://github.com/piecioshka/xhr-sniffer/pull/2/commits/26332ffb9254b59290168553e93771e643f4fd90